.circleci/config.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/circleci-demo-cirpython-django
```

Comment:
working directory name should match your project's name.

Reply:
done.